packages/discord.js/src/structures/Embed.js

```diff
@@ -181,13 +182,7 @@ class Embed {
    * @readonly
    */
   get length() {
-    return (
-      (this.data.title?.length ?? 0) +
-      (this.data.description?.length ?? 0) +
-      (this.data.fields?.reduce((prev, curr) => prev + curr.name.length + curr.value.length, 0) ?? 0) +
-      (this.data.footer?.text.length ?? 0) +
-      (this.data.author?.name.length ?? 0)
-    );
+    return embedLength(this.data);
```

Comment:
imo this alias should be removed and `embedLength` should be used by the user. So this method should be deprecated. 
Also `length` prop in builder does make sense to check length before adding more fields but not not in received data